Tests/TuistDependenciesTests/Cocoapods/CocoaPodsInteractorTests.swift

```diff
@@ -0,0 +1,29 @@
+import ProjectDescription
+import TSCBasic
+import TSCUtility
+import TuistCore
+import TuistGraph
+import TuistSupport
+import XCTest
+
+@testable import TuistDependencies
+@testable import TuistDependenciesTesting
+@testable import TuistSupportTesting
+
+final class CocoaPodsInteractorTests: TuistUnitTestCase {
+    var subject: CocoaPodsInteractor!
+
+    override func setUp() {
+        super.setUp()
+        subject = CocoaPodsInteractor()
+    }
+
+    override func tearDown() {
+        subject = nil
+        super.tearDown()
+    }
+
+    func test_it_runs() throws {
+        try subject.run()
```

Comment:
Need to figure out why this is failing. The execution of the Ruby script works when run from my terminal but fails when run through `System.shared`.